lib/internal/module.js

```diff
@@ -50,6 +51,40 @@ function stripBOM(content) {
   return content;
 }
 
+/**
+ * Find end of shebang line and slice it off
+ */
+function stripShebang(content) {
+  // Remove shebang
+  var contLen = content.length;
+  if (contLen >= 2) {
+    if (content.charCodeAt(0) === 35/*#*/ &&
```

Comment:
Yes it does. This code originates from lib/module.js (see [master branch](https://github.com/nodejs/node/blob/7954d2a4c7f47764643b0941635e093af6170f09/lib/module.js#L543))